.ci.yaml

```diff
@@ -12,12 +12,15 @@ platform_properties:
   linux:
     properties:
       os: Linux
+      signing_certs: "none"
```

Comment:
Why we need a property?

Reply:
For sure just a quick recap on what happened. The initial plan was to use a property to distinguish a code signing bot from a build*-m9 chromium bot. A code signing bot and a m9 bot share the same configs, and a code signing bot might pick up and run post-submit tests if not differentiated. 

There were two approaches on how to add a property to distinguish the bots. 
1. One was to modify ci.yaml directly, either through platform properties or through build properties. 
2. the other was to modify the luci scheduler in common.star in infra, so that the scheduler adds in a default property to each build, and thus change all builds in ci.yaml

The current plan is to use the second approach, because the change required is smaller in scale. We are planning to change infra after this week's flutter 3 release in case it would break anything.

Reply:
Is this because the scheduler does not recognize dimensions and we need to map/translate properties to dimensions?
if this is the case I would suggest adding proper support for dimensions instead of work around it. This will prevent having to change the scheduler/infra-configs and .ci.yaml whenever we need to introduce a new dimension.

Reply:
+1 - This would be a good yak shave. I filed https://github.com/flutter/flutter/issues/103361

Reply:
Right my understanding is that dimensions are the things we pass into luci to tell it which bots should be scheduled to pick up certain builds. And properties are the attributes passed into individual builds and bots, which is related to how the build should be executed.

Oh I didn't know we were trying to circumvent a property/dimension mismatch by supplying dimensions as properties in ci.yaml. Sure I will research and take on https://github.com/flutter/flutter/issues/103361